vignettes/anpl.Rmd

```diff
@@ -1,10 +1,10 @@
 ---
-title: "Vignette Title"
-author: "Vignette Author"
+title: "Adaptive non-parametric learning"
+author: "Simon Lyddon, Miguel Morin, and James Robinson"
 date: "`r Sys.Date()`"
```

Comment:
Should this be today's date?

Reply:
No, the standard is to have the date at which the vignette is run and built, inserted automatically with that command.